test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -223,47 +185,6 @@ if (dialect.startsWith('postgres')) {
         },
       ],
 
-      dropTableQuery: [
-        {
-          arguments: ['myTable'],
-          expectation: 'DROP TABLE IF EXISTS "myTable";',
-        },
-        {
-          arguments: [{ tableName: 'myTable', schema: 'mySchema' }],
-          expectation: 'DROP TABLE IF EXISTS "mySchema"."myTable";',
-        },
-        {
-          arguments: ['myTable', { cascade: true }],
-          expectation: 'DROP TABLE IF EXISTS "myTable" CASCADE;',
-        },
-        {
-          arguments: [{ tableName: 'myTable', schema: 'mySchema' }, { cascade: true }],
-          expectation: 'DROP TABLE IF EXISTS "mySchema"."myTable" CASCADE;',
-        },
-
-        // Variants when quoteIdentifiers is false
```

Comment:
I've not added tests for these since this function is being tested elsewhere and probably will be removed in a future PR. See https://github.com/sequelize/sequelize/pull/14999#discussion_r974055261

Reply:
We should consider writing a PR to print a deprecation warning if `quoteIdentifiers` is set to false

Reply:
For v6? Or deprecate in v7 and remove in v8?

Reply:
I'd say deprecate in v7